api/logs/src/main/java/io/opentelemetry/api/logs/LoggerBuilder.java

```diff
@@ -32,6 +34,14 @@ public interface LoggerBuilder {
    */
   LoggerBuilder setInstrumentationVersion(String instrumentationScopeVersion);
 
+  /**
+   * Specify that emitted log records SHOULD NOT automatically include trace context from the {@link
+   * Context#current()}.
```

Comment:
I wonder if we should say "span context" rather than "trace context" (since I don't think that's a thing outside of the W3C propagator). Also, should we include a link to the `SpanContext` class in the javadoc for completeness?